apps/condo/domains/user/schema/ForgotPasswordAction.js

```diff
@@ -25,6 +25,7 @@ const ForgotPasswordAction = new GQLListSchema('ForgotPasswordAction', {
             schemaDoc: 'Unique token to complete confirmation',
             type: Text,
             isRequired: true,
+            isUnique: true,
```

Comment:
It's already uuid

Reply:
Yes, but on database level it's value is not constrained on uniqueness.
This is correct.